samples/nrf9160/at_cmd_parser/prj.conf

```diff
@@ -0,0 +1,12 @@
+CONFIG_BSD_LIBRARY=y
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+CONFIG_NET_SOCKETS=y
+CONFIG_LOG=y
+CONFIG_LOG_DEFAULT_LEVEL=4
+CONFIG_HEAP_MEM_POOL_SIZE=1024
+CONFIG_MAIN_STACK_SIZE=4096
+
+CONFIG_AT_CMD_INTERFACE=y
+CONFIG_AT_CMD_PARSER=y
```

Comment:
Why do we need both configs here? The parser should be a dependency of the interface.

Reply:
The parser can be used without the interface and the interface can be used without the parser. There is no hard dependency.